packages/cli/internal/pkg/cli/configure_format.go

```diff
@@ -34,7 +34,7 @@ func (o *formatContextOpts) Validate(args []string) error {
 	if len(args) != 1 {
 		return fmt.Errorf("a single format value must be provided")
 	}
-	format := format.FormatterType(o.formatContextVars.format)
+	format := format.FormatterType(args[0])
```

Comment:
Can you please pull the `args[0]` value into a meaningful variable, especially since we reuse it across the code?

Reply:
We aren't actually using `args[0]` within the same function
First we validate the args within `Validate(args)` then we pull out `args[0]` and assign it to opts.format